Buildings/Fluid/Humidifiers/EvaporativeCoolers/Baseclasses/IndirectWetCalculations.mo

```diff
@@ -0,0 +1,178 @@
+within Buildings.Fluid.Humidifiers.EvaporativeCoolers.Baseclasses;
+block IndirectWetCalculations
+  "Calculates the heat transfer in an indirect wet evaporative cooler"
+
+  parameter Real maxEff(
+    displayUnit="1")
+    "Maximum efficiency of heat exchanger coil";
+
+  parameter Real floRat(
+    displayUnit="1")
+    "Coil flow efficency ratio of actual to maximum heat transfer rate";
+
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput VPri_flow(
+    final unit="m3/s",
+    displayUnit="m3/s",
+    final quantity = "VolumeFlowRate")
+    "Primary air volume flow rate"
+    annotation (
+      Placement(
+      visible=true,
+      transformation(
+        origin={-120,-60},
+        extent={{-20,-20},{20,20}},
+        rotation=0),
+      iconTransformation(
+        origin={-140,-60},
+        extent={{-20,-20},{20,20}},
+        rotation=0)));
+
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput VSec_flow(
+    final unit="m3/s",
+    displayUnit="m3/s",
+    final quantity = "VolumeFlowRate")
+    "Secondary air volume flow rate"
+    annotation (
+      Placement(
+      visible=true,
+      transformation(
+        origin={-120,-100},
+        extent={{-20,-20},{20,20}},
+        rotation=0),
+      iconTransformation(
+        origin={-140,-100},
+        extent={{-20,-20},{20,20}},
+        rotation=0)));
+
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput TDryBulPriIn(
+    final unit="K",
+    displayUnit="degC",
+    final quantity="ThermodynamicTemperature")
+    "Dry bulb temperature of the air at the inlet"
+    annotation (Placement(
+      visible=true,
+      transformation(
+        origin={-120,100},
+        extent={{-20,-20},{20,20}},
+        rotation=0),
+      iconTransformation(
+        origin={-140,100},
+        extent={{-20,-20},{20,20}},
+        rotation=0)));
+
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput TWetBulPriIn(
+    final unit="K",
+    displayUnit="degC",
+    final quantity="ThermodynamicTemperature")
+    "Wet bulb temperature of the inlet air"
+    annotation (Placement(
+      visible=true,
+      transformation(
+        origin={-120,60},
+        extent={{-20,-20},{20,20}},
+        rotation=0),
+      iconTransformation(
+        origin={-140,60},
+        extent={{-20,-20},{20,20}},
+        rotation=0)));
+
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput TDryBulSecIn(
+    final unit="K",
+    displayUnit="degC",
+    final quantity="ThermodynamicTemperature")
+    "Dry bulb temperature of the air at the inlet"
```

Comment:
`Dry bulb temperature of the secondary inlet air`

Reply:
Comment updated